src/DIRAC/Core/DISET/private/Transports/SSLTransport.py

```diff
@@ -54,7 +54,13 @@ def checkSanity(urlTuple, kwargs):
     certFile = certTuple[0]
     useCerts = True
   elif "proxyString" in kwargs:
-    if not isinstance(kwargs['proxyString'], six.string_types):
+    if (
+        six.PY2 and not isinstance(
+            kwargs['proxyString'],
+            six.string_types)) or (
+        six.PY3 and not isinstance(
+            kwargs['proxyString'],
+            bytes)):
```

Comment:
Please check this is the same but I think this can be simpler:
```suggestion
    if not isinstance(kwargs['proxyString'], six.string_types if six.PY2 else bytes):
```

Reply:
yes, better thanks !